storybook/stories/API/cartesian/ReferenceLine.stories.tsx

```diff
@@ -47,16 +57,38 @@ const StyleProps: Args = {
   },
   strokeWidth: GeneralStyle.strokeWidth,
   strokeDasharray: GeneralStyle.strokeDasharray,
+  position: {
+    description:
+      "Defines at which position of an axis point the line will start/end if the axis does not have type 'number'.",
+    table: { type: { summary: "'start' | 'middle' | 'end'" }, category: 'Style' },
+  },
+  shape: {
+    description: `If set a ReactElement, the shape of the line can be customized. If set a function, 
+    the function will be called to render a customized shape.`,
+    table: {
+      type: {
+        summary: 'ReactElement | Function',
+      },
+      category: 'Style',
+    },
+  },
+};
+
+const InternalProps: Args = {
```

Comment:
From the type I see further InternalProps, such as 

```
interface InternalReferenceLineProps {
  viewBox?: CartesianViewBox;
  xAxis?: Omit<XAxisProps, 'scale'> & { scale: D3Scale<string | number> };
  yAxis?: Omit<YAxisProps, 'scale'> & { scale: D3Scale<string | number> };
  clipPathId?: number | string;
}
```

Should they also be added?

Reply:
Hi @nikolasrieble, the remaining internal props are added through `ReferenceComponentInternalArgs`